tests/flow/test_variable_length_traversals.py

```
@@ -323,15 +326,15 @@ def test13_fanout(self):
 
         # children of root
         for i in range(3):
-            l = res[i+1][0]
-            identity = res[i+1][1]
+            l = res[i + 1][0]
```

Ruff (0.15.20)

[error] Ambiguous variable name: `l`

(E741)